lib/veri_report.ml

```diff
@@ -0,0 +1,56 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std 
+
+module Map = String.Map
+
+type hist = (string * int) list [@@deriving bin_io, sexp]
+
+type succ = [ 
```

Comment:
The idea was to test a frame and to record it if it's incorrect. so we'll find out what is wrong and how often